particles/Renderer.cpp

```diff
@@ -152,7 +140,7 @@ void Renderer::init(score::gfx::RenderList& renderer)
                 auto bindings = createDefaultBindings(
                             renderer, rt, m_processUBO, m_material.buffer, m_samplers);
                 auto pipeline = buildPipeline(
-                            renderer, mesh, n.m_vertexS, n.m_fragmentS, rt, bindings);
+                            renderer, m_mesh, n.m_vertexS, n.m_fragmentS, rt, bindings);
```

Comment:
Use our mesh member - even if the mesh isn't loaded it's ok, as this function does not need the mesh data, just the "shape" of the mesh which is set in the constructor.
The TexturedParticlesMesh should actually be called TexturedParticlesMeshSpecification as it's not about storing the mesh data, only how it looks  to the gpu